Makefile

```diff
@@ -175,6 +175,12 @@ generate-api-docs: ## Generate api docs
 VENVDIR=$(abspath docs/venv)
 REQUIREMENTS_TXT=docs/requirements.txt
 
+.PHONY: local-docs
+local-docs: venv ## Run mkdocs serve
+	cd docs
+	. $(VENV)/activate; \
+	VENV=$(VENV) docs/scripts/run-mkdocs.sh
```

Comment:
Can't we run `mkdocs serve` from here directly? 🤔  Also, did you get the `serve-docs` target to run?

Reply:
Right, it was leftover, we can run it from main directory :)
I did try but I think for editing locally and then viewing `mkdocs serve` is the better option
It is also explained in the `mike` github repo: https://github.com/jimporter/mike?tab=readme-ov-file#viewing-your-docs